### PR TITLE
Miscellaneous updates to formation-react

### PR DESCRIPTION
**Pull request by cvalarida**
## Description
Just a few minor quality-of-life improvements.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -12,7 +12,6 @@ lerna-debug.log
 node_modules
 npm-debug.log
 package-lock.json
-packages/formation-react/*.js
```
Comment (cvalarida) [inline .gitignore:15]: Added this to `formation-react`'s own `.gitignore`.

File: packages/formation-react/README.md [hunk 2]
```diff
@@ -13,21 +13,22 @@ See [design system](https://department-of-veterans-affairs.github.io/design-syst
 - AdditionalInfo.js
 - AlertBox.js
 - Breadcrumbs.js
-- ExpandingGroup.js
```
Comment (cvalarida) [inline packages/formation-react/README.md:16]: Sorted this list.

File: packages/formation-react/scripts/build.js [hunk 3]
```diff
@@ -49,13 +49,13 @@ const fileNames = [].concat.apply(
   [],
   [
     glob.sync('./src/components/**/*.jsx', {
-      ignore: ['./**/*.unit.spec.jsx', './**/*.stories.(mdx|jsx)'],
+      ignore: ['./**/*.unit.spec.jsx', './**/*.stories.jsx'],
```
Comment (cvalarida) [inline packages/formation-react/scripts/build.js:52]: Stopped `yarn build` from building stories. (Which I thought I had already done, but apparently failed.)

File: packages/formation-react/src/components/ErrorableCheckboxGroup/ErrorableCheckboxGroup.stories.jsx [hunk 4]
```diff
@@ -11,7 +11,11 @@ export default {
   },
 };
 
-const Template = (args) => <ErrorableCheckboxGroup {...args} />;
+const Template = (args) => (
+  <div style={{ paddingLeft: '1em' }}>
+    <ErrorableCheckboxGroup {...args} />
+  </div>
+);
```
Comment (cvalarida) [inline packages/formation-react/src/components/ErrorableCheckboxGroup/ErrorableCheckboxGroup.stories.jsx:18]: Added some padding to the left so we can see the red bar for the error.

File: packages/formation-react/src/components/ErrorableFileInput/ErrorableFileInput.stories.jsx [hunk 5]
```diff
@@ -6,7 +6,11 @@ export default {
   component: ErrorableFileInput,
 };
 
-const Template = (args) => <ErrorableFileInput {...args} />;
+const Template = (args) => (
+  <div style={{ paddingLeft: '1em' }}>
+    <ErrorableFileInput {...args} />
+  </div>
+);
```
Comment (cvalarida) [inline packages/formation-react/src/components/ErrorableFileInput/ErrorableFileInput.stories.jsx:13]: Same here.

**Review by bkjohnson — APPROVED** [review]
Looks good! I wonder if at some point we could/should do something with `preview-body` to give everything more of an inset so that we don't have to have those custom styled wrappers, but that's for another day :upside_down_face:

**Review by U-DON — APPROVED** [review]

File: packages/formation-react/src/components/ErrorableCheckboxGroup/ErrorableCheckboxGroup.stories.jsx (hunk 4 above)
Comment (U-DON) [reply packages/formation-react/src/components/ErrorableCheckboxGroup/ErrorableCheckboxGroup.stories.jsx:18]: I usually opt for `rem` units, but I don't know if we have a standardized unit (and maybe we should).

**Review by Clark-W-Griswold — APPROVED** [review]
👍

File: packages/formation-react/.gitignore [hunk 6]
```diff
@@ -1 +1,2 @@
 yarn.lock
+*.js
```
Comment (U-DON) [inline packages/formation-react/.gitignore:2]: Does this capture the `src/helpers` as well? Those are `.js` files.

Comment (U-DON) [reply packages/formation-react/.gitignore:2]: Maybe it should be something like `/*.js`?

Comment (cvalarida) [reply packages/formation-react/.gitignore:2]: Ooh, good thinking. I just verified this only captures root-level `.js` files by making a change to `helpers/fields.js` and it looks like it worked as expected; the change was tracked.
image

However, _new_ `.js` files didn't get tracked, so changing it to `/*.js` is the way to go. Thanks! :raised_hands:

**Pull request merged**